demos/gallery/samples/chart/InnerTextBar.js

```diff
@@ -0,0 +1,18 @@
+import { Bar } from "@hpcc-js/chart";
+
+new Bar()
+    .innerText({
+        "a": ["some text","lorem ipsum"],
+        "b": ["more text","dolor sit amet"],
+        "c": ["other text","consectetur adipisicing elit"]
+    })
```

Comment:
@jbrundage `innerText` should not needed as the current axis text should just work (we don't have inner text for Pie for example).